lib/core/src/irods_configuration_keywords.cpp

```diff
@@ -166,5 +166,6 @@ namespace irods
 
     // misc. keywords
     const std::string HOSTS_CONFIG_JSON_OBJECT_KW("hosts_config_json_object");
+    const std::string HOSTS_AVAILABLE_RULE_EXECUTORS("available_executors");
```

Comment:
since things are mostly working now - the naming has come back up...

consider...
- `delay_rule_executors`
- `delayed_rule_executors`
- `delay_rule_executor_hostnames`
- `delayed_rule_executor_hostnames`
- `hostnames_to_execute_delay_rules`
- `hostnames_to_execute_delayed_rules`
- `hostnames_for_delay_execution`
- `hostnames_for_delayed_execution`
- `hostnames_for_execution_of_delay_rules`
- `hostnames_for_execution_of_delayed_rules`

Reply:
delay_rule_executors appears to have been selected.